app/DoctrineMigrations/Version20190215035039.php

```diff
@@ -2,6 +2,14 @@
 
 declare(strict_types=1);
 
+/*
+ * This file is part of Eventbase API.
+ *
+ * (c) 2017–2018 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
```

Comment:
We should generally, i.e. in all our projects, consider removing this header or at least remove the the end year from it – or update and maintain it.